### PR TITLE
Improve CLI docs

### PR DESCRIPTION
**Pull request by bszwarc**
**Description**

Changes proposed in this pull request:

- Improve docs

### CODE REVIEW

**Review by clebs — APPROVED** [review]

File: cmd/kyma/provision/aks/cmd.go [hunk 1]
```diff
@@ -32,8 +32,8 @@ func NewCmd(o *Options) *cobra.Command {
 	cmd := &cobra.Command{
 		Use:   "aks",
 		Short: "Provisions an Azure Kubernetes Service (AKS) cluster on Azure.",
-		Long: `Use this command to provision an AKS cluster on Azure for Kyma installation. Use the flags to specify cluster details.
-NOTE: To provision and access the provisioned cluster, make sure you get authenticated by using the Azure CLI. To do so,run ` + "`az login`" + ` and log in with your Azure credentials.`,
+		Long: `Use this command to provision an AKS cluster on Azure for Kyma installation. Use the flags to specify cluster details. 
+	NOTE: To provision and access the provisioned cluster, make sure you get authenticated by using the Azure CLI. To do so,run ` + "`az login`" + ` and log in with your Azure credentials.`,
```
Comment (klaudiagrz) [inline cmd/kyma/provision/aks/cmd.go:36]:
```suggestion
	NOTE: To provision and access the provisioned cluster, make sure you get authenticated by using the Azure CLI. To do so, run ` + "`az login`" + ` and log in with your Azure credentials.`,
```

**Review by clebs — APPROVED** [review]

File: docs/03-01-use-kyma-cli.md [hunk 2]
```diff
@@ -26,6 +26,7 @@ See [the full list of commands and flags](/cli/commands/).
 |--------------------|----------------|---------------|---------|
 | [`completion`](/cli/commands#kyma-completion-kyma-completion)| None| Generates and displays the bash or zsh completion script. | `kyma completion`|
 | [`console`](/cli/commands#kyma-console-kyma-console)| None| Launches Kyma Console in a browser window. | `kyma console` |
+| [`create`](/cli/commands/#kyma-create-kyma-create)|[`system`](cli/commands/#kyma-create-system-kyma-create-system)| Creates resources on the Kyma cluster. **NOTE:** The `kyma create` and `kyma create system` commands are still in alpha version. | `kyma create` |
```
Comment (klaudiagrz) [inline docs/03-01-use-kyma-cli.md:29]:
```suggestion
| [`create`](/cli/commands/#kyma-create-kyma-create)|[`system`](cli/commands/#kyma-create-system-kyma-create-system)| Creates resources on the Kyma cluster. **NOTE:** The `kyma create` and `kyma create system` commands are still in the alpha version. | `kyma create` | 
```

File: docs/gen-docs/kyma_provision_aks.md [hunk 3]
```diff
@@ -6,8 +6,8 @@ Provisions an Azure Kubernetes Service (AKS) cluster on Azure.
 
 ## Synopsis
 
-Use this command to provision an AKS cluster on Azure for Kyma installation. Use the flags to specify cluster details.
-NOTE: To provision and access the provisioned cluster, make sure you get authenticated by using the Azure CLI. To do so,run `az login` and log in with your Azure credentials.
+Use this command to provision an AKS cluster on Azure for Kyma installation. Use the flags to specify cluster details. 
+	NOTE: To provision and access the provisioned cluster, make sure you get authenticated by using the Azure CLI. To do so,run `az login` and log in with your Azure credentials.
```
Comment (klaudiagrz) [inline docs/gen-docs/kyma_provision_aks.md:10]:
```suggestion
	NOTE: To provision and access the provisioned cluster, make sure you get authenticated by using the Azure CLI. To do so, run `az login` and log in with your Azure credentials.
```

Comment (bszwarc) [reply docs/gen-docs/kyma_provision_aks.md:10]: same as above :)

**Review by klaudiagrz — APPROVED** [review]

**Pull request merged**